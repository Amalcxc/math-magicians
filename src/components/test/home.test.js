import Home from '../home'
import React from 'react';
import { render, screen } from '@testing-library/react';

test('renders to home', () => {
  render(<Home />);
  const linkElement = screen.getByText('Welcome to the Home');
  expect(linkElement).toBeInTheDocument();
});
