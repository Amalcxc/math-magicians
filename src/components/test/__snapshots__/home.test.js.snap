// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to home 1`] = `
<h2>
  Welcome to the Home
</h2>
`;
